numpy/_core/_printoptions.py

```diff
@@ -0,0 +1,22 @@
+import sys
```

Comment:
This file is already in a private module, why should it have an underscore in its name? Instead, I'd add a docstring explaining why it exists, and perhaps even define an `__all__ = ["format_options"]`

Reply:
We could also put this into `np._globals` (that way it is clearly out of the way of any core import, I am not sure if it might otherwise get into problems if some place imports `_arrayprint` directly or so).  I agree with no need for nested underscores.